sdk/metricsadvisor/Azure.AI.MetricsAdvisor/CHANGELOG.md

```diff
@@ -74,6 +78,7 @@
 - The class `DatasourceCredential` (now called `DataSourceCredentialEntity`) is now abstract.
 - `AlertQueryTimeMode` and `FeedbackQueryTimeMode` are now regular enums.
 - The enum `AuthenticationType` present in some `DataFeedSource` subtypes is now an extensible enum.
+- Removed constructor `(string workspaceId, string query)` from the `LogAnalyticsDataFeedSource`.
```

Comment:
Not actually supported by the service. Other properties are required now. We'll possibly add this constructor back again as new types of credentials are added.